src/linagora.esn.videoconference/app/videoconference.run.js

```diff
@@ -0,0 +1,7 @@
+(function(angular) {
+  'use strict';
+
+  angular.module('linagora.esn.videoconference')
+    .run();
```

Comment:
run what?

Reply:
My mistake, I didn't check all code.
https://ci.linagora.com/linagora/lgs/openpaas/linagora.esn.videoconference/blob/master/frontend/app/videoconference.run.js => It seems that the original file doesn't run anything